File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Weathers/Description.java

package com.appsbygu.kadai_weather_java.models.apis.Weathers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Description extends RealmObject {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("publicTime")
    @Expose
    private String publicTime;

    public String getText() {
        return text;
    }

    public String getPublicTime() {
        return publicTime;
    }


}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/weathers/Description.kt
package com.appsbygu.kadai_weather_java.models.apis.weathers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

open class Description : RealmObject() {

    @SerializedName("text")
    @Expose
    var text: String? = null

    @SerializedName("publicTime")
    @Expose
    var publicTime: String? = null


}
